src/o365/spfx/commands/project/project-externalize/model/IVisitationResult.ts

```diff
@@ -0,0 +1,7 @@
+import { FileEditSuggestion } from "./FileEditSuggestion";
```

Comment:
The interface looks good but I wouldn't put it in model which is meant for classes and interfaces that represent a SharePoint Framework project and its pieces.

Reply:
moved to the rules folder as it's an "internal" data structure